_episodes/16-parallel.md

```diff
@@ -564,10 +459,9 @@ and resubmit the job each time.
 You can also increase the number of CPUs.
 
 * How good is the value for &#960;?
-* How much memory did it need?
 * How long did the job take to run?
 
-## How Much Does MPI Improve Performance?
+## How Much Does Distributed Memory Parallelism Improve Performance?
```

Comment:
I guess MPI is inherently Distributed Memory Parallelism, but if we're going to use the latter term, may want to describe/define it earlier in the lesson.

Reply:
Ok.

Reply:
I'm not sure we have a lot to gain by replacing a term the learners likely are somewhat familiar with but perhaps didn't fully understand with one they probably haven't heard of (until now) and don't understand. Keeping the more familiar term is less effort for them.

Reply:
The motivation is that there maybe other programming models, such as Dask and Spark which run on an HPC cluster and use a submission system but do not use MPI. However, it is probably better to make such a change in another pull request if these can be incorporated as optional paths/sections of HPC intro.

Reply:
Ok, reverted to MPI.